apiary.apib

```diff
@@ -422,9 +422,12 @@ We consider the following changes to be backwards-incompatible:
 We list all backwards-compatible additions here. These are currently available in all published versions.
 (There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
 
+### May 2023
+- We added `team_lead_id` to the filters on `teams.list`.
+
 #### April 2023
```

Comment:
Shouldn't this be

```suggestion
### April 2023
```

?

Reply:
https://github.com/teamleadercrm/api/pull/692